settings.py

```diff
@@ -114,6 +114,12 @@
     'less#1.7.3',
     'backbone#0.9.1',
     'bootstrap-daterangepicker#2.1.13',
+    'd3#3.1.5',
```

Comment:
I may have removed a `d3#3.0` in the rebase. Not sure if those were both intended to be loaded, or if that was a mistake and they can all use the same version